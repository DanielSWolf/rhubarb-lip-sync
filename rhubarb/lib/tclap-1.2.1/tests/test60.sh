#!/bin/sh

# failure
../examples/test9 -VVV -N --noise -rr > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test60.out; then
	exit 0
else
	exit 1
fi

