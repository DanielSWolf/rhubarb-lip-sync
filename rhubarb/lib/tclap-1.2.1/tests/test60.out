PARSE ERROR: Argument: -r (--reverse)
             Argument already set!


USAGE: 

   ../examples/test9  [-N] ...  [-V] ...  [-r] [--] <won't see this>


Where: 

   -N,  --noise  (accepted multiple times)
     Level of noise

   -V,  --verbose  (accepted multiple times)
     Level of verbosity

   -r,  --reverse
     REVERSE instead of FORWARDS

   --,  --ignore_rest
     Ignores the rest of the labeled arguments following this flag.

   <won't see this>
     a random word


   Command description message

